examples/display_usb_hid/main/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 description: BSP Display USB HID Example
 dependencies:
-  idf: ">=4.4"
-  usb_host_hid: "*"
-  esp32_s3_usb_otg:
-    version: "*"
-    override_path: "../../../bsp/esp32_s3_usb_otg"
+  esp-box-3:
+    version: '*'
+    override_path: ../../../bsp/esp-box-3
```

Comment:
We did not test this example with esp-box-3, are you sure it works OK? The users will need a extension board with USB A connector to make it work, right?